src/main/java/leo/task/Task.java

```diff
@@ -0,0 +1,201 @@
+package leo.task;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+
+import leo.ui.Ui;
+
+public class Task implements Serializable {
+    private static final long serialVersionUID = 1L;
+    
+    enum Type {
+        TODO, DEADLINE, EVENT
+    };
+
+    private String taskDesc;
+    private boolean isDone = false;
+    private Type type;
+    public final static List<String> descCommands = Arrays
+            .asList(new String[] { "todo", "deadline", "event", "mark", "unmark", "delete" });
+
+    public final static List<String> commands = Arrays
```

Comment:
Similarly, for commands I think it should be all caps?

`Variable 'commands' should be in ALL_CAPS (if it is a constant) or be private (otherwise). [ConstantName]`

Reply:
Thanks for raising this up. Accidentally changed the access modifiers while implementing more classes.